client/wfm/src/service/WfmService.ts

```diff
@@ -22,6 +22,7 @@ export interface CompleteStepParams {
   stepCode: string;
 }
 
+// FIXME adjust UI to not relay on summary and pick all data from Workorder
```

Comment:
By the way, apart from the `nextStepIndex` logic, the updated workorder should simply have all the data that was in the summary before.

Reply:
nextStepIndex logic will be part of the workoder - Let's cht about it.